src/redux/actions/congress.js

```diff
@@ -22,6 +22,7 @@ export const {
   renounceCandidacy,
   unapproveTreasurySpend,
   voteAtMotions,
+  citizenProposeRepealLegislation,
```

Comment:
This has nothing to do with congress. It should be either `democracy` or `legislation` actions (put it wherever we already have the action for proposing new legislation)